rts/Map/SMF/ROAM/RoamMeshDrawer.h

```diff
@@ -77,6 +77,9 @@ class CRoamMeshDrawer : public IMeshDrawer, public CEventClient
 
 	float3 lastCamPos[MESH_COUNT];
 
+	int numPatchesLeftVisibility[MESH_COUNT] = {0};
+	int tesselationsSinceLastReset[MESH_COUNT] = {0};
```

Comment:
don't think 0 is necessary here

Reply:
are you sure?

Reply:
In C++ `{}` zero-initializes arrays.

https://godbolt.org/z/EFMBbW 

`{ 0 } ` says "initialize the first element to 0, and zero initialize the other elements", which just opens things to confusion and may lead people into thinking all values are initialized to what is in the braces.

Which it is not: https://godbolt.org/z/FzirNb

Reply:
So in my case, just use {} instead of {0} ?

Reply:
yes